src/gui/accountsettings.cpp

```diff
@@ -227,7 +227,7 @@ QString AccountSettings::selectedFolderAlias() const
 
 void AccountSettings::slotOpenAccountWizard()
 {
-    OwncloudSetupWizard::runWizard(qApp,  SLOT(slotownCloudWizardDone(int)), this);
+    ocApp()->gui()->runNewAccountWizard();
```

Comment:
Not a fan of getter chains and especially not on singletons - but this PR gets rid of another singleton, so I guess it's still an improvement :-D